Ginger/Ginger/ValueExpression/ValueExpressionEditorPage.xaml.cs

```diff
@@ -575,16 +662,18 @@ private void InsertNewVarTreeItem(TreeViewItem parentTvi, VariableBase vb)
         private void InsertAddNewVarTreeItem(TreeViewItem parentTvi, eVariablesLevel varLevel)
         {
             TreeViewItem newVarTvi = new TreeViewItem();
-            SetItemView(newVarTvi, "Add New String " + GingerDicser.GetTermResValue(eTermResKey.Variable) , varLevel, "@Add_16x16.png");
+            SetItemView(newVarTvi, "Add New String " + GingerDicser.GetTermResValue(eTermResKey.Variable) , varLevel, eImageType.Add);
             parentTvi.Items.Add(newVarTvi);
             newVarTvi.MouseDoubleClick += tviAddNewVarTreeItem_MouseDoubleClick;
         }
 
         private void AddDataSources()
         {
+            TreeViewItem Parent = AddOrGetCategory("Data");
+            //TreeViewItem child = AddOrGetSubCategory("Data Source", Parent);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6356765)